build.sh

```diff
@@ -38,17 +38,17 @@ TEST_REPO=git@github.com:rdkcentral/ut-core.git
 # This function checks the latest version of UT core and recommends an upgrade if reuqired
 function check_next_revision()
 {
-    pushd ./ut-core
+    pushd ./ut-core > /dev/null
     # Set default UT_CORE_PROJECT_VERSION to next revision, if it's set then we don't need to tell you again
     if [ -v ${UT_CORE_PROJECT_VERSION} ]; then
-        UT_CORE_PROJECT_VERSION=$(git tag | grep ${UT_PROJECT_MAJOR_VERSION} | sort -r | head -n1)
+        UT_CORE_PROJECT_VERSION=$(git tag | grep ^${UT_PROJECT_MAJOR_VERSION} | sort -r | head -n1)
         UT_NEXT_VERSION=$(git tag | sort -r | head -n1)
         echo -e ${YELLOW}ut-core version selected:[${UT_CORE_PROJECT_VERSION}]${NC}
         if [ "${UT_NEXT_VERSION}" != "${UT_CORE_PROJECT_VERSION}" ]; then
             echo -e ${RED}--- New Version of ut-core released [${UT_NEXT_VERSION}] consider upgrading ---${NC}
         fi
     fi
-    popd
+    popd > /dev/null
 }
 
 # Check if the common document configuration is present, if not clone it
```

Comment:
Good to have .clone_complete hidden file after cloning is complete/d rather the current way of looking at the directory on line number: 55

Reply:
updated to re-word to ut-core, rather than common documentation